static/specification.html

```diff
@@ -110,16 +110,88 @@ <h3>
             &lt;link
               rel="monetization"
               href="https://example.com/pay"
-              onmonetization="sayThanks()"&gt;
+              onmonetization="sayThanks(this)"&gt;
 
             &lt;script&gt;
-              function sayThanks(event) {
+              function sayThanks(monetizedLink) {
                 // Do something here
               }
             &lt;/script&gt;
           </pre>
         </aside>
       </section>
+      <section>
+        <h3>
+          Monetization events
+        </h3>
+        <p>
+          The {{MonetizationEvent}} DOM events provide information such as the
+          amount received, the currency of the payment (what we call the "asset
+          code"), and a receipt that can be used to verify the receipt of
+          payment. As in the previous example above, these events are
+          dispatched to [^link^] elements and bubble up the DOM tree.
+        </p>
+        <p>
+          To listen for {{MonetizationEvent}} events, you need to add an
+          [=event listener=] to the the relevant [^link^] element via
+          JavaScript:
+        </p>
+        <aside class="example">
+          <pre class="html">
+            &lt;link rel="monetization" href="https://example.com/pay"&gt;
+            &lt;script&gt;
+              const link = document.querySelector('link[rel="monetization"]');
+              link.addEventListener("monetization", event =&gt; {
+                // See how much your received and in what currency
+                const { amount, assetCode, assetScale } = event;
+                console.log(`Received ${assetCode}${amount} (${assetScale}).`);
+
+                // And use the receipt to verify the payment using
+                // a payment verification service (see next example).
+                verifyReceipt(event);
+              });
+            &lt;/script&gt;
+          </pre>
+        </aside>
+      </section>
+      <section>
+        <h3>
+          Verifying a receipt
+        </h3>
+        <p>
+          The {{MonetizationEvent}}'s {{MonetizationEvent/receipt}} attribute
+          is a unique string can be used to verify the payment using a payment
+          verification service.
+        </p>
+        <p>
+          Below is a hypothetical verification service "verifier.example" that
+          performs verification via a `POST` HTTP request.
+        </p>
+        <aside class="example">
+          <pre class="JS">
+            /** @type {MonetizationEvent} event */
+            async function verifyReceipt(event) {
+              const body = JSON.stringify({
+                receipt: event.receipt
+              });
+
+              const response = await fetch('https://verifier.example/verify/', {
```

Comment:
@wilsonianb 
Generally we'll have the paymentPointer encoded in the url here right ?

Reply:
Maybe the example should have something with `encodeURIComponent(event.paymentPointer)`

Reply:
right, you mean if we send it as URL parameter? If yes:

```JS
const url = new URL('https://verifier.example/verify/');
url.searchParams.set("verify", event.paymentPointer);
fetch(url, ...);
```

Reply:
yeah, just some way of encoding the pp into the url
tbh, I'm not sure if it's strictly needed

Brandon ? 
@wilsonianb